client/src/layouts/Footer/Footer.jsx

```diff
@@ -61,18 +64,16 @@ function Footer({ isMainPage, handleNotification, count, map, handleDrawer, setM
         <COVIDBanner />
       ) : (
         <IconsContainer>
-          {onlyShowIcon !== "addHero" &&
-            onlyShowIcon !== "lightCandle" && (
-              <Numbers>
-                <NumbersText>{count}</NumbersText>
-                <Text>Candles Lit</Text>
-              </Numbers>
-            ) /* This is just a filler until it gets hooked up as well; */}
+          {onlyShowIcon !== "addHero" && onlyShowIcon !== "lightCandle" && (
+            <FooterButton handleClick={handleBack}>Candles lit</FooterButton>
+          ) /* This is just a filler until it gets hooked up as well; */}
           {onlyShowIcon !== "candlesLit" && onlyShowIcon !== "addHero" && (
             <CandleWrapper addNotification={handleNotification} setOnlyShowIcon={setOnlyShowIcon} />
           )}
           {onlyShowIcon !== "candlesLit" && onlyShowIcon !== "lightCandle" && (
-            <AddHeroButton handleClick={setOnlyShowIcon} />
+            <FooterButton icon={addHeroIcon} handleClick={() => setOnlyShowIcon("addHero")}>
+              add Hero
```

Comment:
capitalize first letter. Alternatively we can do capitlization using css of `FooterButton`.